common/bitcointree/psbt.go

```diff
@@ -0,0 +1,59 @@
+package bitcointree
+
+import (
+	"bytes"
+
+	"github.com/btcsuite/btcd/btcutil/psbt"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+)
+
+var (
+	COSIGNER_PSBT_KEY_PREFIX = []byte("cosigner")
```

Comment:
whats this

Reply:
It lets you write the cosigners public keys in the PSBTs. These keys have been used to create the aggregated musig2 key signing the tree txs. All cosigners need them to sign the tree. 

Note that now, the cosigners keys are only owned by the ASP so later, when we'll add client-side signing, we would need to add extra data (the aggregated nonce)